### PR TITLE
Issue#100

### PR DESCRIPTION
**Pull request by irina060981**
1) Created the component - wordForms - it generates the following form "targetWord" (forms) for the inflections component

2) Forms are created in the component based on the language model direction

3) Added unit tests for the wordForms component

### CODE REVIEW

File: src/vue-components/wordforms.vue [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          return this.defineFormsBySelectedView()
+        }
+      }
+    },
+    methods: {
+      defineFormsBySelectedView: function () {
+        let forms = new Set()
+        for (let lexeme of this.lexemes) {
+          for (let inflection of lexeme.inflections) {
+          	if (inflection['part of speech'].values.includes(this.partOfSpeech)) {
```
Comment (kirlat) [inline src/vue-components/wordforms.vue]: I think it's better to use `Feature.types.part` instead of `'part of speech'` here

Comment (balmas) [reply src/vue-components/wordforms.vue]: agree

File: src/vue-components/inflections.vue [hunk 2]
```diff
@@ -182,8 +189,10 @@
       },
       forms: function () {
         let forms = []
-        if (this.selectedView && this.selectedView.forms) {
+        if (this.selectedView && this.selectedView.forms && this.selectedView.forms.values.length > 0) {
```
Comment (balmas) [inline src/vue-components/inflections.vue]: Can we get rid of this method here now? I think it isn't used anymore correct?

File: tests/vue-components/wordforms.test.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            }
+          ]
+        }]
+      }
+    })
+
+    expect(cmp.vm.forms).toEqual(['fooform'])
+  })
+
+  it('5 WordForms - render (no part of speach)', () => {
```
Comment (balmas) [inline tests/vue-components/wordforms.test.js]: Can we just rename the test actually to "5 WordForms - render only forms with matching part of speech" ?

File: src/vue-components/wordforms.vue (hunk 1 above)
Comment (irina060981) [reply src/vue-components/wordforms.vue]: Done

File: src/vue-components/inflections.vue (hunk 2 above)
Comment (irina060981) [reply src/vue-components/inflections.vue]: Oh, sorry :) I have missed it. Done.

File: tests/vue-components/wordforms.test.js (hunk 3 above)
Comment (irina060981) [reply tests/vue-components/wordforms.test.js]: Done.

**Pull request merged**